tdesign/src/Layout/BaseLayout/BaseLayout.tsx

```diff
@@ -15,7 +15,7 @@ export interface BaseLayoutProps extends Omit<ILogoProps, "linkTo"> {
   children?: React.ReactNode;
   renderHeaderRight?: () => React.ReactNode;
   renderHeaderCenter?: () => React.ReactNode;
-  extraHeaderStyle?: SxProps<Theme>;
+  extraHeaderStyle?: object;
```

Comment:
Why do we go to a less specific type?

Reply:
It looks like @onpaws made this to match the convention of `extraHeader` below it, which was added 9 months ago when switching from `theme-ui`.

Agreed definitely doesn't make sense, I'll try switching it and see if there is some issue.